### DISCUSSION TITLE
Add the possibility of setting the gamut of relative color notations

### DISCUSSION THREAD

**Discussion by diogoterremoto** [evilmartians/oklch-picker#163] [General]
I know very little of color theory but I think I found a useful feature to add to this tool.

For example, I often receive colors in RGB from designers. However, the designer’s app is set to display P3 colors instead of the standard sRGB.

This creates an issue because this tool currently interprets RGB values as being in the sRGB color space by default. The RGB color `(255,0,0)` in **P3** is different (much more _vivid_) than `(255,0,0)` than **sRGB**, although the values of red, green, and blue are the same.

To work around this, I use the macOS ColorSync Utility calculator to convert 8-bit RGB colors in P3 into floating-point RGB in P3. After that, I use the `display-p3` CSS notation in this tool to get the corresponding OKLCH color value. This allows me to maintain color accuracy across different spaces.

Is adding the possibility of specified the gamut of relative color notations (like Hex and RGB) a good feature? Is there any other way to do this more efficiently?

**Comment by ai** [comment]
To change RGB value from sRGB to P3, you need to change output format to Figma P3

![изображение](https://github.com/user-attachments/assets/24ac9271-5299-4a55-be9f-6b04ec544884)

A small thread about Figma P3 in my Twitter https://x.com/sitnikcode/status/1688969741916942336
